### PR TITLE
javascript/javascript2/week1

### PR DESCRIPTION
**Pull request by somayehdvr**

### CODE REVIEW

File: javascript/javascript2/week1/animalName.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const animalNames = [
+    'Contemplative Teal Beetle',
+    'Unwavering White Ridley',
+    'Sensible Fuchsia Mockingbird',
+    'Steady Plum Boa',
+    'Convivial Purple Cottontail',
```
Comment (obelitz) [inline javascript/javascript2/week1/animalName.js:6]: Oh damn, you serving us them fancy words 😄

File: javascript/javascript2/week1/animalName.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    'Bald Forest Deer',
+    'Silky Lake Owl',
+    'Fat Sea Snake',
+    'Scaled Cave Zebra',
+    'Juicy Glue Gun Kitty'
+];
+
+const spiritAnimal = document.querySelector('.spiritAnimal');
+const button = document.querySelector(".submit");
+button.innerHTML = "submit";
```
Comment (obelitz) [inline javascript/javascript2/week1/animalName.js:16]: This seems like something I'd set directly in the HTML.

File: javascript/javascript2/week1/animalName.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+
+function myFunction() {
```
Comment (obelitz) [inline javascript/javascript2/week1/animalName.js]: Not a descriptive name. 
`generateSpiritName` maybe.

File: javascript/javascript2/week1/danishLetters.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+//Find and count the Danish letters
+///////////////////////////////////////////////
```
Comment (obelitz) [inline javascript/javascript2/week1/danishLetters.js:2]: I like the visuals here.

File: javascript/javascript2/week1/danishLetters.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    const danishLetters = ["æ", "ø", "å"]
+    const count = {total: 0}
+    for (const letter of string.split("")) {
+        if (danishLetters.includes(letter)) {
+            if (!count[letter]) {
+                count[letter]= 0
+            }
+            count.total++
+            count[letter]++
+        }
```
Comment (obelitz) [inline javascript/javascript2/week1/danishLetters.js:13]: Good solution!
Did you work on this with @Jul-S?

File: javascript/javascript2/week1/exercices.js [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+        button2.innerHTML = "change color"
+        body.style.backgroundColor = "white"
+        body.style.color = "blue"
+    }
+}
+
+button2.addEventListener("click", myFunction2)
+
+////////////////////////////////////////////////
+console.log(body)
```
Comment (obelitz) [inline javascript/javascript2/week1/exercices.js]: I'm not sure what this file is? Is it homework?

File: javascript/javascript2/week1/exercices.js [hunk 7]
```diff
@@ -0,0 +1,91 @@
+//Favorite dishes
+/////////////////////////////////////////////////
+const favoriteDishes = ["pizza", "Ghorme", "gheime"]
+let ul = document.createElement('ul');
+let body = document.querySelector('body');
```
Comment (obelitz) [inline javascript/javascript2/week1/exercices.js]: These should be const.

File: javascript/javascript2/week1/exercices.js [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+//Podcast
+/////////////////////////////////////////////////
+const podcasts = [{
+    name: 'The mystery om of Johan Klausen Varbourg',
+    imageUrl: 'https://picsum.photos/536/354'
+},
+{
+    name: 'Tips about dogs with small legs',
```
Comment (obelitz) [inline javascript/javascript2/week1/exercices.js:21]: Cute!

File: javascript/javascript2/week1/exercices.js [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+ul = document.createElement('ul');
+body.appendChild(ul)
+
+for (let i = 0; i < podcasts.length; i++){
+    let li = document.createElement('li');
+    h1 = document.createElement('h1');
+    h1.innerHTML = podcasts[i].name;
+    li.appendChild(h1);
+
+    if (podcasts[i].imageUrl){
```
Comment (obelitz) [inline javascript/javascript2/week1/exercices.js:42]: Nice!

File: javascript/javascript2/week1/exercices.js [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+    elementToAppendImageTo.appendChild(image);
+}
+
+insertImage('https://picsum.photos/536/354', document.querySelector('body'));
+
+//Simple eventlistener
+/////////////////////////////////////////////////
+let button = document.querySelector(".btn-class1")
+
+function myFunction () {
```
Comment (obelitz) [inline javascript/javascript2/week1/exercices.js]: Still not a descriptive name.

File: javascript/javascript2/week1/exercises.html [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+        <link rel="stylesheet" href="style.css">
+        
+    </head>
+    <body>
+        <h1> js 2 w 1</h1>
+
+        <div class="events"></div>
+        <div id="logo"></div>
+        <button class="btn-class1">click me</button>
+        <button class="btn-class2">change color</button>
```
Comment (obelitz) [inline javascript/javascript2/week1/exercises.html:14]: These class-names should be more descriptive. Eg. eventlistener-button and color-change-button

File: javascript/javascript2/week1/shortestWord.js [hunk 12]
```diff
@@ -0,0 +1,15 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+function findTheShortestWord (arrayOfWords) {
+    let shortestWord = arrayOfWords[0];
+
+    for (let word of arrayOfWords){
+        if (word.length < shortestWord.length) {
+            shortestWord = word;
+        }
+    }
```
Comment (obelitz) [inline javascript/javascript2/week1/shortestWord.js:10]: Short and sweet :)

**Review by obelitz — COMMENTED** [review]
Good job!

Notes:
- There are some of the exercises that I don't recognize
- You should work on naming. Functions, classes, variables

File: javascript/javascript2/week1/animalName.js (hunk 2 above)
Comment (somayehdvr) [reply javascript/javascript2/week1/animalName.js:16]: I thought we should do them in JS

File: javascript/javascript2/week1/animalName.js (hunk 3 above)
Comment (somayehdvr) [reply javascript/javascript2/week1/animalName.js]: yes I forgot to change the name

File: javascript/javascript2/week1/danishLetters.js (hunk 5 above)
Comment (somayehdvr) [reply javascript/javascript2/week1/danishLetters.js:13]: thanks, no.

File: javascript/javascript2/week1/exercices.js (hunk 7 above)
Comment (somayehdvr) [reply javascript/javascript2/week1/exercices.js]: yes you're right

File: javascript/javascript2/week1/exercices.js (hunk 8 above)
Comment (somayehdvr) [reply javascript/javascript2/week1/exercices.js:21]: :)

File: javascript/javascript2/week1/exercices.js (hunk 9 above)
Comment (somayehdvr) [reply javascript/javascript2/week1/exercices.js:42]: thanks

File: javascript/javascript2/week1/exercices.js (hunk 6 above)
Comment (somayehdvr) [reply javascript/javascript2/week1/exercices.js]: no it was for my reference :)

File: javascript/javascript2/week1/exercises.html (hunk 11 above)
Comment (somayehdvr) [reply javascript/javascript2/week1/exercises.html:14]: yes it was for practice and test

File: javascript/javascript2/week1/shortestWord.js (hunk 12 above)
Comment (somayehdvr) [reply javascript/javascript2/week1/shortestWord.js:10]: thanks

File: javascript/javascript2/week1/animalName.js (hunk 2 above)
Comment (obelitz) [reply javascript/javascript2/week1/animalName.js:16]: It's probably good training!

**Pull request merged**